sphinx/builders/linkcheck.py

```diff
@@ -722,6 +722,8 @@ def handle_starttag(self, tag: Any, attrs: Any) -> None:
 def _allowed_redirect(
     url: str, new_url: str, allowed_redirects: dict[re.Pattern[str], re.Pattern[str]]
 ) -> bool:
+    if allowed_redirects is False:
+        return False
```

Comment:
We can remove this in accompaniament with the undo suggested at: https://github.com/sphinx-doc/sphinx/pull/13483/files#r2120568655

```suggestion
```

Reply:
(done)